### ISSUE TITLE
[Feature Request]: show if video is saved (and maybe in what playlist)

### ISSUE THREAD

**Issue by MrHappy** [FreeTubeApp/FreeTube#4982]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

Now when I watch a video I cannot see if the video is already saved in a playlist (and in what playlist).

### Proposed Solution

In the previous version there was a yellow asterisk to indicate it was saved.
Something like that and (preferably the name of the playlist).

### Alternatives Considered

Saving it again and after that checking other playlist if I did not save it there already and remove duplicates.

### Issue Labels

display more information to user

**Comment by jasonhenriquez** [comment]
Can you clarify what change is wanted and where? Like the "Add to Playlist" `+` icon having a number attached to it? Or some type of change on the prompt where you add the video to different playlists?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @MrHappy,

> In the previous version there was a yellow asterisk to indicate it was saved.

This isnt gone but it could be that you havent enabled this yet.
Go into one of your playlists and click on the following button

![Capture](https://github.com/FreeTubeApp/FreeTube/assets/73130443/3e6cdfa4-a948-4b43-b060-60344862e0ea)

Now you will see a Clock icon instead of a Star icon on the places you are used to, when hovering on a thumbnail and under the video player.

When clicked on the icon it will save that video to the playlist you have enabled this feature for.


https://github.com/FreeTubeApp/FreeTube/assets/73130443/29917e89-94a8-482f-91bc-e49d42bc5ea4



We also indicate if a video is already added to a playlist


https://github.com/FreeTubeApp/FreeTube/assets/73130443/c4d8f383-5a5c-41d8-b5a8-278ed79411ca

Closing this issue as i think this has addressed your points.

**Comment by toby63** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc I don't understand why you are against re-implementing a simple indicator to the video page?
I guess that many people see this as a regression.
Instead of seeing it immediately, you have to click on "add to playlist" and then see it in a seperate menu, it is now two steps, where it was one step in the old versions.

@jasonhenriquez I think a number would be useful indeed, but a colored + icon would also be a good start.

**Comment by jasonhenriquez** [comment]
> @efb4f5ff-1298-471a-8973-3d47447115dc I don't understand why you are against re-implementing a simple indicator to the video page?

They didn't say that they were opposed to that; it was more than the issue description didn't specify where the indication was wanted.

> @jasonhenriquez I think a number would be useful indeed, but a colored + icon would also be a good start.

I'm not sure if color alone would be an intuitive indicator of that. Would you be open to creating a sketch or wiremock of your recommendation?

**Comment by jasonhenriquez** [comment]
I don't think this is as salient given that the quick bookmark icon resolves this issue for the most common use case of one "main" playlist where people put most videos into, but I will reopen this issue as this seems to be a novel request.

The main question that I have is what use cases we're trying to solve for. 
- Is it that you want to know if there exists a playlist containing this video, even if it's not in the quick bookmark playlist?
  - If so, we could theoretically add some visual identifier to the Add to Playlist icon to represent this.
- Do you need to know which exact playlists have which video from the icon alone? 
  - In which case, we could theoretically have a shortlist of those playlists appear on hover of the "Add to Playlist" icon.
- Do you want to know the number of playlists that have this video, so that you can use the Add to Playlist prompt to determine your next steps?
  - We could theoretically put a number inside or next to the "Add to Playlist" icon to signify that.

So it would need to be known what is the actual issue that needs to be fixed, and how can it be done in a way that doesn't notably complicate the user experience for people unconcerned with this use case.

**Comment by MrHappy** [comment]
> Hi @MrHappy,
> 
> > In the previous version there was a yellow asterisk to indicate it was saved.
> 
> This isnt gone but it could be that you havent enabled this yet. Go into one of your playlists and click on the following button
> 
> ![Capture](https://private-user-images.githubusercontent.com/73130443/324979253-3e6cdfa4-a948-4b43-b060-60344862e0ea.PNG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQwMzM0MDIsIm5iZiI6MTcxNDAzMzEwMiwicGF0aCI6Ii83MzEzMDQ0My8zMjQ5NzkyNTMtM2U2Y2RmYTQtYTk0OC00YjQzLWIwNjAtNjAzNDQ4NjJlMGVhLlBORz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI1VDA4MTgyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQzNDQ4NWU3YjcyZDM4OWJlYmZmMGFhNDA0NmU3ZDhjNDg2ZWI4YTFjOTljYjlmZDdlZjRhOTFkZTFiNzI5NDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.662LpxL0hoEVSekhBWeqrJAnPbgfDOYZFU9ZGrpGMLM)
> 
> Now you will see a Clock icon instead of a Star icon on the places you are used to, when hovering on a thumbnail and under the video player.
> 
> When clicked on the icon it will save that video to the playlist you have enabled this feature for.
> VirtualBoxVM_uxVAdHcxFE.mp4
> 
> We also indicate if a video is already added to a playlist
> VirtualBoxVM_Ge4sMkAMTQ.mp4
> 
> Closing this issue as i think this has addressed your points.

I did not have this enabled, but unfortunately this only works for the playlist I marked as 'favorite'.
If I save to a different playlist it does not show anything.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> but unfortunately this only works for the playlist I marked as 'favorite'.
If I save to a different playlist it does not show anything.

Yes thats the intended behavior. It only shows a checkmark on the thumbnail if it is saved to the quick bookmark playlist. 

If you want to know if a video is saved in a playlist that is NOT the quick bookmark playlist you have to go to the add to playlist window for more information.

**Comment by qupear** [comment]
> Is it that you want to know if there exists a playlist containing this video, even if it's not in the quick bookmark playlist?
> 
>     * If so, we could theoretically add some visual identifier to the Add to Playlist icon to represent this.

This is exactly what is a very wanted feature. Just any indicator that video is any playlist. Speaking about me personally, I don't need any counters here, or name of exact playlist. I use just four themed playlists and I am 100% sure in which playlist a video could be. The only question is a video already saved or not. And currently the only way to find this out is to manually scrolling and remembering. Indicator could be some dot, or color, or frame, or icon.

**Comment by MrHappy** [comment]
I was not aware another response was needed. Please reopen.
The option offered by @qupear seems to fit my needs.
